checkstyle/suppressions.xml

```diff
@@ -8,5 +8,7 @@
     <suppress checks="JavadocStyle" files=".+Test\.java" />
     <suppress checks="JavadocMethod" files=".+Test\.java" />
     <suppress files="[\\/]generated[\\/]" checks="[a-zA-Z0-9]*"/>
+    <suppress files="\.pem" checks="[a-zA-Z0-9]*"/>
```

Comment:
These should not be checked in.

Reply:
This is for ease of use for standalone deployments.